src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceVersion2.java

```diff
@@ -287,6 +295,66 @@ public Response uploadIrbDocument(
     }
   }
 
+  @POST
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/progress_report/{parentReferenceId}")
+  @RolesAllowed({RESEARCHER})
+  public Response postProgressReport(
+      @Auth AuthUser authUser,
+      @PathParam("parentReferenceId") String parentReferenceId,
+      @FormDataParam("dar") String dar,
+      @FormDataParam("collaboratorRequiredFile") InputStream collabInputStream,
+      @FormDataParam("collaboratorRequiredFile") FormDataContentDisposition collabFileDetails,
+      @FormDataParam("ethicsApprovalRequiredFile") InputStream ethicsInputStream,
+      @FormDataParam("ethicsApprovalRequiredFile") FormDataContentDisposition ethicsFileDetails) {
+    User user = userService.findUserByEmail(authUser.getEmail());
+    DataAccessRequest parentDar = dataAccessRequestService.findByReferenceId(parentReferenceId);
+    checkAuthorizedUpdateUser(user, parentDar);
+    DataAccessRequest payload = populateDarFromJsonString(user, dar);
```

Comment:
Passing in malformed json throws an unhelpful error, maybe we can wrap some of this to return a better (400) error:
```
{
  "code": 500,
  "message": "There was an error processing your request. It has been logged (ID 2c75f2b5d758abe2)."
}
```